docs/products/kafka/concepts/horizontal-vertical-scaling.rst

```diff
@@ -0,0 +1,50 @@
+Scaling options in Apache Kafka
+===============================
+
+Aiven for Apache Kafka has a number of predefined plans that specify the
+number of brokers and the capacity of individual brokers. The predefined plans consist of 3, 6, 9, 15, or 30 brokers, but we can also create
+larger custom plans based on customer requirements.
+
+To increase the capacity of an existing Kafka cluster, two options are
+available:
+
+* **Vertically** scaling the performance of individual brokers
```

Comment:
I would use nouns "Vertical scaling"/"Horizontal scaling" instead of adverbs. Sounds a bit more consistent as you state them as "options", which is a noun as well

Reply:
yep, thanks!